### PR TITLE
Translation/from en to it

### PR DESCRIPTION
**Pull request by RaydenMK**
@swirle13
 Latest version of the file with added rows has been translated.

### CODE REVIEW

File: src/locales/it/translation.json [hunk 1]
```diff
@@ -328,129 +328,129 @@
...
+                        "Questa è una soluzione per chiarire quali siano le carte dell'Espansione \"Avventure\". Queste si distinguono per il simbolo con un piccola stella in alto al centro del lato anteriore della carta."
                     ]
                 }
             ]
         },
-        "card": {
-            "numInPack": "# in this pack",
-            "numSoFar": "# so far",
-            "numTotal": "# total"
+         "card": {
```
Comment (swirle13) [inline src/locales/it/translation.json:430]:
```suggestion
        "card": {
```

This accidentally had an additional space included, which will prevent the json from being valid and working properly.

File: src/locales/it/translation.json [hunk 2]
```diff
@@ -328,129 +328,129 @@
...
+            "Questa è una prima bozza per far arrivare il Controllo Componenti nelle mani della seconda ondata di spedizioni il più velocemente possibile e, come tale, sarà minimale e (probabilmente) privo di traduzione.",
+            "Il mio obiettivo è aggiungere diverse sezioni per i segnalini e le miniature subito dopo la pubblicazione.",
+            "Al momento le spunte di controllo non sono permanenti. Se aggiorni o esci da questa pagina perderai i tuoi progressi. Mi scuso per l'inconveniente mentre risolvo la questione."
         ],
-        "legendHeader": "Legend",
+        "legendHeader": "Legenda",
         "legendBody": {
-            "header": "If two columns: read left to right, top to bottom. This will ensure the running totals of cards spread across multiple packs (e.g. \"Monster cards: 11 (11/29)\") are read properly.",
+            "header": "Se su due colonne: leggi da sinistra a destra, dall'alto in basso. Ciò garantirà che i totali parziali delle carte distribuite su più mazzi (ad esempio \"Carte Mostro: 11 (29/11)\") vengano letti correttamente.",
```
Comment (swirle13) [inline src/locales/it/translation.json:396]: Does italian count items out of a total number of items in a flipped order, e.g. "29 total, 11 so far"?

The value here, "11/29", is equal to "11 out of the total 29 items so far". I'm assuming this was an accidental recognition of this being a date, November 29, and flipping it to be accurate for Italian, do I have that correct?

File: src/locales/it/translation.json [hunk 3]
```diff
@@ -462,154 +462,154 @@
...
+        "Lost Mount": "Destriero Perduto",
+        "Monster card (Troll)": "Carta Mostro (Troll)",
+        "Trophy cards": "Carte Trofeo",
+        "Bomb cards": "Carte Bomba",
+        "Starting Action cards (Manticore)": "Carte Azione Iniziali (Manticora)",
+        "Advanced Action cards": "Carte Azione Avanzata",
+        "Starting Action cards (Ciri)": "Carte Azione Iniziali (Ciri)",
+        "Unique Action cards (witchers)": "Carte Azione Uniche (Witchers)",
+        "Large Monster cards": "Carte grandi Mostro",
         "Difficulty Jumbo Reference cards": "Difficulty Jumbo Reference cards",
```
Comment (swirle13) [inline src/locales/it/translation.json:500]: Do you need to translate this line as well?

File: src/locales/it/translation.json [hunk 4]
```diff
@@ -462,154 +462,154 @@
...
         ],
         "github": "GitHub",
-        "projectTimeline": "Project Timeline",
+        "projectTimeline": "Cronologia del progetto",
         "githubLinks": [
             {
-                "text": "Find a bug?",
-                "link": "https://github.com/swirle13/The-Witcher-Old-World-Tool/issues/new?assignees=&labels=bug&projects=&template=bug_report.md&title=%5BBUG%5D+"
+                "text": "Hai trovato un bug?",
+                "link": "https://github.com/swirle13/The-Witcher-Old-World-Tool-Issues/issues/new?assignees=&labels=bug&projects=&template=bug_report.md&title=%5BBUG%5D+"
```
Comment (swirle13) [inline src/locales/it/translation.json:574]: can you update this to the previous version? I have updated this url to consolidate to the primary repo now that it is public.

```suggestion
                "link": "https://github.com/swirle13/The-Witcher-Old-World-Tool/issues/new?assignees=&labels=bug&projects=&template=bug_report.md&title=%5BBUG%5D+"
```

File: src/locales/it/translation.json [hunk 5]
```diff
@@ -462,154 +462,154 @@
...
             },
             {
-                "text": "Enhancement idea?",
-                "link": "https://github.com/swirle13/The-Witcher-Old-World-Tool/issues/new?assignees=&labels=enhancement&projects=&template=feature_request.md&title=%5BENHANCEMENT%5D+"
+                "text": "Suggerisci un miglioramento",
+                "link": "https://github.com/swirle13/The-Witcher-Old-World-Tool-Issues/issues/new?assignees=&labels=enhancement&projects=&template=feature_request.md&title=%5BENHANCEMENT%5D+"
```
Comment (swirle13) [inline src/locales/it/translation.json:578]:
```suggestion
                "link": "https://github.com/swirle13/The-Witcher-Old-World-Tool/issues/new?assignees=&labels=enhancement&projects=&template=feature_request.md&title=%5BENHANCEMENT%5D+"
```

File: src/locales/it/translation.json [hunk 6]
```diff
@@ -462,154 +462,154 @@
...
             },
             {
-                "text": "Discussions",
-                "link": "https://github.com/swirle13/The-Witcher-Old-World-Tool/discussions"
+                "text": "Discussioni",
+                "link": "https://github.com/swirle13/The-Witcher-Old-World-Tool-Issues/discussions"
```
Comment (swirle13) [inline src/locales/it/translation.json:582]:
```suggestion
                "link": "https://github.com/swirle13/The-Witcher-Old-World-Tool/discussions"
```

File: src/locales/it/translation.json [hunk 7]
```diff
@@ -462,154 +462,154 @@
...
             {
-                "text": "Task Board",
+                "text": "Board delle attività",
                 "link": "https://github.com/users/swirle13/projects/2/views/1"
             },
             {
-                "text": "All open bugs/enhancements",
-                "link": "https://github.com/swirle13/The-Witcher-Old-World-Tool/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc"
+                "text": "Tutti i bugs riscontrati/miglioramenti proposti",
+                "link": "https://github.com/swirle13/The-Witcher-Old-World-Tool-Issues/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc"
```
Comment (swirle13) [inline src/locales/it/translation.json:594]:
```suggestion
                "link": "https://github.com/swirle13/The-Witcher-Old-World-Tool/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc"
```

**Review by swirle13 — CHANGES_REQUESTED** [review]
So very good, thank you so much. Just a few really small things and it should all be good to merge!

**Comment by RaydenMK** [comment]
> So very good, thank you so much. Just a few really small things and it should all be good to merge!

@swirle13 
I did the corrections requested. Do you think it can be merged?

**Review by swirle13 — APPROVED** [review]
Looks good, tested with no issues on my local machine!

**Pull request merged**